cg/utils/email.py

```diff
@@ -0,0 +1,21 @@
+from smtplib import SMTP
+
+from cg.exc import EmailNotSentError
+from cg.models.email import EmailInfo
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+
+
+def send_mail(email_info: EmailInfo):
+    """Email handler. Sending new user request email to admin"""
+
+    print(email_info.json())
+    with SMTP(email_info.smtp_server) as server:
+        msg = MIMEMultipart()
```

Comment:
Won't work without server.connect() or at least didn't when I tried

Reply:
Fixed. Email working + fixed bug "add file to hk"